### DISCUSSION TITLE
Refreshing map file in the app

### DISCUSSION THREAD

**Discussion by martinsimsa** [OpenOrienteering/mapper#2178] [Questions & Answers] [answered]
Hi, is there any way how to refresh a map file while in mapper? I would like to edit the map file and see the results instantly in the app. Now I have to close the window and open it once again. Thanks

**Comment by dg0yt** [comment, marked as answer]
Closing and reopening the map is the only way.

**Reply by martinsimsa** [reply]
Okay thanks
